Build error when building against Electron 7 on Linux

<!--

Have you read Atom's Code of Conduct? By filing an Issue, you are expected to comply with it, including treating everyone with respect: https://github.com/atom/.github/blob/master/CODE_OF_CONDUCT.md

Do you want to ask a question? Are you looking for support? The Atom message board is the best place for getting support: https://discuss.atom.io

-->

### Prerequisites

* [ ] Put an X between the brackets on this line if you have done all of the following:
    * Reproduced the problem in Safe Mode: <https://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode>
    * Followed all applicable steps in the debugging guide: <https://flight-manual.atom.io/hacking-atom/sections/debugging/>
    * Checked the FAQs on the message board for common solutions: <https://discuss.atom.io/c/faq>
    * Checked that your issue isn't already filed: <https://github.com/issues?utf8=✓&q=is%3Aissue+user%3Aatom>
    * Checked that there is not already an Atom package that provides the described functionality: <https://atom.io/packages>

### Description

<!-- Description of the issue -->

Compilation error:

```pre
../src/scrollbar-style-observer-non-mac.cc: In static member function ‘static void ScrollbarStyleObserver::Init(v8::Local<v8::Object>)’:
../src/scrollbar-style-observer-non-mac.cc:35:51: error: no matching function for call to ‘v8::Object::Set(v8::Local<v8::String>, v8::Local<v8::Function>)’
     Nan::GetFunction(newTemplate).ToLocalChecked());
                                                   ^
```

See: https://dev.azure.com/DeeDeeG/b/_build/results?buildId=886&view=logs&j=0da5d1d9-276d-5173-c4c4-9d4d4ed14fdb&t=cccdf131-0893-5d7b-ccea-9ccfc8953f95&l=889

### Steps to Reproduce

1. Modify Atom `master` branch to depend on to depend on this package, version 3.2.2
2. Modify Atom `master` branch to use Electron version 7.3.3
3. On Linux, try to build Atom

**Expected behavior:**

<!-- What you expect to happen -->

Atom should build successfully

**Actual behavior:**

<!-- What actually happens -->

`scrollbar-style` fails to compile during Atom's bootstrap process

**Reproduces how often:**

<!-- What percentage of the time does it reproduce? -->

100%

### Versions

<!-- You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running. -->

Info from (unaffected, succesfully built) macOS app:

```console
% ~/atom/out/Atom\ Dev.app/Contents/MacOS/Atom\ Dev --version   
Atom    : 1.55.0-dev-689759bb5
Electron: 7.3.3
Chrome  : 78.0.3904.130
Node    : 12.8.1
```

```console
% ~/atom/out/Atom\ Dev.app/Contents/Resources/app/apm/bin/apm --version
apm  2.5.2-atomic.3.1
npm  6.14.9
node 12.4.0 x64
atom 1.55.0-dev-689759bb5
python 2.7.16
git 2.27.0
```

Reproduced on OS: Ubuntu 16.04 (CI) and Ubuntu 20.04 (not CI)

### Additional Information

<!-- Any additional information, configuration or data that might be necessary to reproduce the issue. -->

Seems like there is a NAN issue that is triggered on non-mac OSes, perhaps just on Linux.

--- a/src/scrollbar-style-observer-non-mac.cc
+++ b/src/scrollbar-style-observer-non-mac.cc
@@ -30,7 +30,9 @@ void ScrollbarStyleObserver::Init(Local<Object> target) {
   newTemplate->InstanceTemplate()->SetInternalFieldCount(1);
   Local<ObjectTemplate> proto = newTemplate->PrototypeTemplate();
   Nan::SetMethod(proto, "getPreferredScrollbarStyle", ScrollbarStyleObserver::GetPreferredScrollbarStyle);
+  v8::Local<v8::Context> context = Nan::GetCurrentContext();
   target->Set(
+    context,
     Nan::New<String>("ScrollbarStyleObserver").ToLocalChecked(),
     Nan::GetFunction(newTemplate).ToLocalChecked());
 }

